app/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -68,17 +71,18 @@
         android:layout_marginEnd="24dp"
         android:ems="10"
         android:inputType="phone"
-        android:hint="Phone"
+        android:hint="@string/edit_profile_phone"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/edit_text_edit_profile_email" />
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_edit_profile_email"
+        android:autofillHints="" />
 
     <EditText
         android:id="@+id/edit_text_edit_profile_description"
-        android:hint="Description"
+        android:hint="@string/edit_profile_description"
         android:inputType="textMultiLine"
         android:imeOptions="flagNoExtractUi"
-        android:gravity="top|left"
+        android:gravity="top|start"
```

Comment:
Yes, it should be more logical this way. But with "top|left", as I implemented it in the first weeks on the project, I didn't notice any strange appearance... did you?

Reply:
No, this change was not to correct a strange appearance, it was just to follow android studio tips and "rules" which puts "top" with "start".  There are several similar changes in my modifications.